src/services/refactor.service.js

```diff
@@ -0,0 +1,9 @@
+export const refactorPhoneNumber = (phone = '') => {
+  if (typeof phone !== 'string') return '';
+  return (
+    `+${phone.slice(0, 3)}(${phone.slice(3, 5)})-${phone.slice(
+      5,
```

Comment:
Convert it to one line, please, because it will increase the readability of the formatting

Reply:
converted